app/ui/src/app/app.component.html

```diff
@@ -203,37 +207,37 @@
           <span class="pficon pficon-help"
                 data-toggle="tooltip"
                 title="User"></span>
-          <span class="list-group-item-value">Help</span>
+          <span class="list-group-item-value">{{ 'help' | synI18n }}</span>
         </a>
         <div class="nav-pf-secondary-nav">
           <div class="nav-item-pf-header">
             <a role="button" class="secondary-collapse-toggle-pf" data-toggle="collapse-secondary-nav"></a>
-            <span>Help</span>
+            <span>{{ 'help' | synI18n }}</span>
           </div>
           <ul class="list-group">
             <li class="list-group-item">
               <a [href]="tutorialLink"
                 rel="nofollow"
                 role="link"
                 target="_blank"
-                class="list-group-item-value">Sample Integration Tutorials</a>
+                class="list-group-item-value">{{ 'header.helpmenu.sampletutorials' | synI18n }}</a>
             </li>
             <li class="list-group-item">
               <a [href]="userGuideLink"
                 rel="nofollow"
                 role="link"
                 target="_blank"
-                class="list-group-item-value">User Guide</a>
+                class="list-group-item-value">{{ 'header.helpmenu.userguide' | synI18n }}</a>
             </li>
             <li class="list-group-item">
-              <a routerLink="/support" role="link"  class="list-group-item-value">Support</a>
+              <a routerLink="/support" role="link"  class="list-group-item-value">{{ 'header.helpmenu.support' | synI18n }}</a>
             </li>
             <li class="list-group-item">
               <a href="mailto:fuse-ignite-tech-preview@redhat.com"
                 rel="nofollow"
                 role="link"
                 target="_blank"
-                class="list-group-item-value">Contact Us</a>
+                class="list-group-item-value">{{ 'header.helpmenu.contactus' | synI18n }}</a>
```

Comment:
This could be reused elsewhere in the future. Perhaps another good candidate for the `shared` group.